configs/default/TMTR-TMOTORVELOXF7V2.config

```diff
@@ -0,0 +1,124 @@
+# Betaflight / TMOTORVELOXF7V2 (TMR7) 4.3.1 Sep 23 2022 / 17:19:06 () MSP API: 1.44
+
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_DPS310
```

Comment:
Please remove BARO - if not physical present on board.

Reply:
FC has DPS310 on board.